en/intro_to_command_line/README.md

```diff
@@ -153,14 +153,14 @@ You'll probably see something similar on your machine. Once you open the command
 
 Many commands you can type at the command prompt have built-in help that you can display and read! For example, to learn more about the current directory command:
 
-<!--sec data-title="Command help: OS X and Linux" data-id="OSX_Linux_pwd" data-collapse=true ces-->
+<!--sec data-title="Command help: OS X and Linux" data-id="OSX_Linux_pwd1" data-collapse=true ces-->
```

Comment:
Shouldn't this become `OSX_Linux_man` (and `Windows_syntax_help` or something similar)? After all I think the point here is that `man` and `/?` can be used to get more insight into almost any program/command.

Reply:
I haven't check what's the best id would be here (I just make them unique, treating this pull request as an emergency one, because we have online verison not working properly).

Moreover, we already have `OSX_Linux_pwd`/`windows_cd` twice, `OSX_Linux_pwd2`/`windows_cd2` and `OSX_Linux_pwd3`/`windows_cd3`.

But sure, I will check how we should name this parts of chapter. Give me a moment.

Reply:
@patrys Done!